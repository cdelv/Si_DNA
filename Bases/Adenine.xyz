15
XYZ
H            17.242291520    11.438884040     9.854683000
N            16.314673100    11.946352510     9.854683000
Si           14.723042700    11.114009900     9.854683000
H            14.779137100     9.618864990     9.854683000
N            13.283021810    12.033110750     9.854683000
Si           13.835541960    13.754848040     9.854682990
Si           13.489342380    15.945712390     9.854683010
N            11.952980190    16.798198710     9.854683000
H            11.905344200    17.850573110     9.854683000
H            11.022238220    16.305746630     9.854683000
N            14.878005650    16.981520520     9.854683000
Si           16.531917950    16.518552190     9.854682990
H            17.487914990    17.677179260     9.854683000
N            17.308063940    14.960690030     9.854683000
Si           16.051835250    13.760987270     9.854683000
