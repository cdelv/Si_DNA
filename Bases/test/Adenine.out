Siesta Version  : v4.1.5-1-g384057250
Architecture    : gcc
Compiler version: GNU Fortran (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
Compiler flags  : mpifort -O2 -fPIC -ftree-vectorize -fallow-argument-mismatch
PP flags        : -DFC_HAVE_ABORT -DMPI -DSIESTA__FLOOK -DSIESTA__METIS -DSIESTA__MUMPS -DCDF -DNCDF -DNCDF_4 -DNCDF_PARALLEL -DSIESTA__MRRR -DSIESTA__ELPA
Libraries       : libncdf.a libfdict.a  -lflookall -ldl -lnetcdff -lnetcdf -lhdf5_hl -lhdf5 -lhdf5_fortran -lz -lscalapack-openmpi -lopenblas -lmetis -lzmumps -lcmumps -ldmumps -lmumps_common -lpord -lsmumps -lesmumps -lscotch -lscotcherr -fopenmp -lelpa
PARALLEL version
NetCDF support
NetCDF-4 support
NetCDF-4 MPI-IO support
METIS ordering support
Lua support

* Running on 4 nodes in parallel
>> Start of run:   1-JUN-2023  14:22:30

                           ***********************       
                           *  WELCOME TO SIESTA  *       
                           ***********************       

reinit: Reading from standard input
reinit: Dumped input in INPUT_TMP.08750
************************** Dump of input data file ****************************
# mpirun -np 4 siesta-4.1.5 < Adenine.fdf > Adenine.out
SystemLabel      Adenine
NumberOfAtoms    15
NumberOfSpecies  3
%block ChemicalSpeciesLabel
    1    1  H
    2    7  N
    3   14  Si
%endblock ChemicalSpeciesLabel
############################################
# This overrides Defaults.fdf configuration
############################################
MD.TypeOfRun                CG # CG, Verlet
MD.Steps                    0
AtomicCoordinatesFormat NotScaledCartesianAng
%block AtomicCoordinatesAndAtomicSpecies
    2.46315852   -5.19921296    0.00000000   1       1  H
    1.53554010   -4.69174449   -0.00000000   2       2  N
   -0.05609030   -5.52408710    0.00000000   3       3  Si
    0.00000410   -7.01923201   -0.00000000   1       4  H
   -1.49611119   -4.60498625    0.00000000   2       5  N
   -0.94359104   -2.88324896   -0.00000001   3       6  Si
   -1.28979062   -0.69238461    0.00000001   3       7  Si
   -2.82615281    0.16010171    0.00000000   2       8  N
   -2.87378880    1.21247611   -0.00000000   1       9  H
   -3.75689478   -0.33235037    0.00000000   1      10  H
    0.09887265    0.34342352   -0.00000000   2      11  N
    1.75278495   -0.11954481   -0.00000001   3      12  Si
    2.70878199    1.03908226    0.00000000   1      13  H
    2.52893094   -1.67740697    0.00000000   2      14  N
    1.27270225   -2.87710973   -0.00000000   3      15  Si
%endblock AtomicCoordinatesAndAtomicSpecies
#############
# Unit Cell
#############
#LatticeConstant 1.0 Ang
#%block LatticeVectors
#    14.779133    0.000000    0.000000
#    0.000000   16.638097    0.000000
#    0.000000    0.000000    9.854683
#%endblock LatticeVectors
%include ../../Defaults.fdf
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: Adenine
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    1 Label: H
Species number:   2 Atomic number:    7 Label: N
Species number:   3 Atomic number:   14 Label: Si

Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from H.psf

Pseudopotential generated from a relativistic atomic calculation

Valence configuration for pseudopotential generation:
1s( 1.00) rc: 0.01
2p( 0.00) rc: 0.60
3d( 0.00) rc: 1.99
4f( 0.00) rc: 2.49
Ground state valence configuration:   2s02  2p03
Reading pseudopotential information in formatted form from N.psf

Pseudopotential generated from a relativistic atomic calculation

Valence configuration for pseudopotential generation:
2s( 2.00) rc: 0.94
2p( 3.00) rc: 0.59
3d( 0.00) rc: 3.46
4f( 0.00) rc: 2.47
Ground state valence configuration:   3s02  3p02
Reading pseudopotential information in formatted form from Si.psf

Pseudopotential generated from a relativistic atomic calculation

Valence configuration for pseudopotential generation:
3s( 2.00) rc: 1.67
3p( 2.00) rc: 2.04
3d( 0.00) rc: 2.89
4f( 0.00) rc: 2.49
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For N, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For Si, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0100        Charge= 0.17977+309
Lmxo=0 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=2  polorb=1  (1s)
            splnorm:   0.62000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for H                     (Z =   1)

read_vps: Pseudopotential generation method:
read_vps: ATM4.2.7  Troullier-Martins                       
Valence charge for ps generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  0.0098
V l=1 = -2*Zval/r beyond r=  0.5817
V l=2 = -2*Zval/r beyond r=  1.9372
All V_l potentials equal beyond r=  1.9372
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.9372

VLOCAL1: 99.0% of the norm of Vloc inside     11.567 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     26.362 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    2.25106
atom: Maximum radius for r*vlocal+2*Zval:    1.91306
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  2.036604   el= -0.477180   Ekb= -6.081590   kbcos= -0.277285
   l= 1   rc=  2.195416   el=  0.001076   Ekb= -0.926943   kbcos= -0.034643
   l= 2   rc=  2.251056   el=  0.002010   Ekb= -0.130941   kbcos= -0.009788

KBgen: Total number of Kleinman-Bylander projectors:    9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.002000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    6.518340
                 energy =   -0.475305
                kinetic =    0.912139
    potential(screened) =   -1.387444
       potential(ionic) =   -1.907295

   izeta = 2
                 rmatch =    2.141150
              splitnorm =    0.620000
                 energy =    0.566467
                kinetic =    3.772856
    potential(screened) =   -3.206389
       potential(ionic) =   -3.759093

POLgen: Perturbative polarization orbital with L=  1

POLgen: Polarization orbital for state 1s

   izeta = 1
                     rc =    6.518340
                 energy =    0.533861
                kinetic =    1.121086
    potential(screened) =   -0.587225
       potential(ionic) =   -1.053027
atom: Total number of Sankey-type orbitals:  5

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.518340

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
N                    Z=   7    Mass=  14.010        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=0  (2s)
            splnorm:   0.28000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=2  polorb=1  (2p)
            splnorm:   0.28000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for N                     (Z =   7)

read_vps: Pseudopotential generation method:
read_vps: ATM4.2.7  Troullier-Martins                       
Valence charge for ps generation:    5.00000

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  1.2889
V l=1 = -2*Zval/r beyond r=  1.2889
V l=2 = -2*Zval/r beyond r=  3.4178
V l=3 = -2*Zval/r beyond r=  2.4084
All V_l potentials equal beyond r=  3.3753
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  3.4178

VLOCAL1: 99.0% of the norm of Vloc inside      3.810 Ry
VLOCAL1: 99.9% of the norm of Vloc inside      8.683 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    4.02088
atom: Maximum radius for r*vlocal+2*Zval:    3.54838
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  3.417763   el= -1.365098   Ekb= -4.433130   kbcos= -0.707855
   l= 1   rc=  3.375302   el= -0.521201   Ekb=-13.264039   kbcos= -0.535594
   l= 2   rc=  3.872880   el=  0.001991   Ekb= -0.641094   kbcos= -0.062456
   l= 3   rc=  3.970931   el=  0.003093   Ekb= -0.474517   kbcos= -0.018236

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.002000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    4.730425
                 energy =   -1.363399
                kinetic =    1.245246
    potential(screened) =   -2.608645
       potential(ionic) =   -8.121105

   izeta = 2
                 rmatch =    2.531849
              splitnorm =    0.280000
                 energy =   -0.959908
                kinetic =    2.409435
    potential(screened) =   -3.369343
       potential(ionic) =   -9.427975

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.002000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    5.924108
                 energy =   -0.519262
                kinetic =    3.672940
    potential(screened) =   -4.192202
       potential(ionic) =   -9.518354

   izeta = 2
                 rmatch =    2.531849
              splitnorm =    0.280000
                 energy =   -0.104258
                kinetic =    6.807538
    potential(screened) =   -6.911796
       potential(ionic) =  -13.149920

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 2p

   izeta = 1
                     rc =    5.924108
                 energy =    1.427508
                kinetic =    2.872379
    potential(screened) =   -1.444871
       potential(ionic) =   -5.563644
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 3.00)                                                            
Vna: chval, zval:    5.00000   5.00000

Vna:  Cut-off radius for the neutral-atom potential:   5.924108

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
Si                   Z=  14    Mass=  28.090        Charge= 0.17977+309
Lmxo=1 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=0  (3s)
            splnorm:   0.28000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=3
          i=1  nzeta=2  polorb=1  (3p)
            splnorm:   0.28000    
               vcte:    0.0000    
               rinn:    0.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000      0.0000    
            lambdas:    1.0000      1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

atom: Called for Si                    (Z =  14)

read_vps: Pseudopotential generation method:
read_vps: ATM4.2.7  Troullier-Martins                       
Valence charge for ps generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
V l=0 = -2*Zval/r beyond r=  2.7826
V l=1 = -2*Zval/r beyond r=  2.7826
V l=2 = -2*Zval/r beyond r=  2.9253
V l=3 = -2*Zval/r beyond r=  2.7826
All V_l potentials equal beyond r=  2.8530
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.9253

VLOCAL1: 99.0% of the norm of Vloc inside      5.333 Ry
VLOCAL1: 99.9% of the norm of Vloc inside     12.154 Ry
atom: Maximum radius for 4*pi*r*r*local-pseudopot. charge    3.39869
atom: Maximum radius for r*vlocal+2*Zval:    2.96205
GHOST: No ghost state for L =  0
GHOST: No ghost state for L =  1
GHOST: No ghost state for L =  2
GHOST: No ghost state for L =  3

KBgen: Kleinman-Bylander projectors: 
   l= 0   rc=  3.075247   el= -0.794092   Ekb=  5.879390   kbcos=  0.167461
   l= 1   rc=  3.153101   el= -0.300068   Ekb=  2.642817   kbcos=  0.058301
   l= 2   rc=  3.273594   el=  0.001958   Ekb= -1.185135   kbcos= -0.104411
   l= 3   rc=  3.232927   el=  0.003046   Ekb= -0.497174   kbcos= -0.015889

KBgen: Total number of Kleinman-Bylander projectors:   16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.002000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    6.429618
                 energy =   -0.792283
                kinetic =    0.525293
    potential(screened) =   -1.317576
       potential(ionic) =   -3.744923

   izeta = 2
                 rmatch =    3.998416
              splitnorm =    0.280000
                 energy =   -0.583081
                kinetic =    1.069570
    potential(screened) =   -1.652651
       potential(ionic) =   -4.235185

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.002000 Ry

   izeta = 1
                 lambda =    1.000000
                     rc =    8.464845
                 energy =   -0.298236
                kinetic =    0.785360
    potential(screened) =   -1.083596
       potential(ionic) =   -3.273779

   izeta = 2
                 rmatch =    4.587816
              splitnorm =    0.280000
                 energy =   -0.131871
                kinetic =    1.403687
    potential(screened) =   -1.535557
       potential(ionic) =   -4.015310

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 3p

   izeta = 1
                     rc =    8.464845
                 energy =    0.321949
                kinetic =    1.055677
    potential(screened) =   -0.733728
       potential(ionic) =   -2.672950
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00)                                                            
 3p( 2.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   8.464845

atom: _________________________________________________________________________

prinput: Basis input ----------------------------------------------------------

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    1 H                       # Species index, atomic number, species label
    2    7 N                       # Species index, atomic number, species label
    3   14 Si                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
H                     1                    # Species label, number of l-shells
 n=1   0   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.518      2.141   
   1.000      1.000   
N                     2                    # Species label, number of l-shells
 n=2   0   2                         # n, l, Nzeta 
   4.730      2.532   
   1.000      1.000   
 n=2   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   5.924      2.532   
   1.000      1.000   
Si                    2                    # Species label, number of l-shells
 n=3   0   2                         # n, l, Nzeta 
   6.430      3.998   
   1.000      1.000   
 n=3   1   2 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   8.465      4.588   
   1.000      1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file H.ion.nc
Dumping basis to NetCDF file N.ion.nc
Dumping basis to NetCDF file Si.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      4.65470  -9.82509   0.00000  1        1
siesta:      2.90175  -8.86612  -0.00000  2        2
siesta:     -0.10600 -10.43902   0.00000  3        3
siesta:      0.00001 -13.26443  -0.00000  1        4
siesta:     -2.82724  -8.70217   0.00000  2        5
siesta:     -1.78313  -5.44855  -0.00000  3        6
siesta:     -2.43735  -1.30842   0.00000  3        7
siesta:     -5.34066   0.30255   0.00000  2        8
siesta:     -5.43068   2.29125  -0.00000  1        9
siesta:     -7.09951  -0.62805   0.00000  1       10
siesta:      0.18684   0.64898  -0.00000  2       11
siesta:      3.31228  -0.22591  -0.00000  3       12
siesta:      5.11886   1.96358   0.00000  1       13
siesta:      4.77899  -3.16984   0.00000  2       14
siesta:      2.40506  -5.43695  -0.00000  3       15

siesta: Automatic unit cell vectors (Ang):
siesta:   14.782281    0.000000    0.000000
siesta:    0.000000   16.643462    0.000000
siesta:    0.000000    0.000000    9.854683

siesta: System type = molecule  

initatomlists: Number of atoms, orbitals, and projectors:     15   155   205

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = spin-orbit
redata: Number of spin components                   = 8
redata: Time-Reversal Symmetry                      = F
redata: Spin spiral                                 = F
############################################################
This spin-orbit implementation uses a local approximation.
From 4.2 and onwards the full non-local approximation is implemented
You are strongly advised to use >=4.2 versions!
############################################################
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        3
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = Atomic and Orbital charges
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   340.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        1
redata: Max. number of SCF Iter                     =      200
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000008
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.000001 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9978 K
redata: Fix the spin of the system                  =   F
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 6
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
mix.SCF:    Restart steps                        = 24
mix.SCF:    Restart save steps                   = 3
mix.SCF: Spin-component mixing                   all
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 6
  restart 24
  restart.save 3
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    50.000000
Total ionic charge:    50.000000

* ProcessorY, Blocksize:    2  24


* Orbital distribution balance (max,min):    48    24


Time-reversal symmetry not used.

siesta: k-grid: Number of k-points =     1
siesta: k-grid: Cutoff (effective) =     4.927 Ang
siesta: k-grid: Supercell and displacements
siesta: k-grid:    1   0   0      0.000
siesta: k-grid:    0   1   0      0.000
siesta: k-grid:    0   0   1      0.000

diag: Algorithm                                     = ELPA-1stage
diag: Parallel over k                               =   F
diag: Use parallel 2D distribution                  =   T
diag: Parallel block-size                           = 24
diag: Parallel distribution                         =     2 x     2
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcoor: Atomic coordinates (Ang):                          
    2.46315852   -5.19921296    0.00000000   1       1  H
    1.53554010   -4.69174449    0.00000000   2       2  N
   -0.05609030   -5.52408710    0.00000000   3       3  Si
    0.00000410   -7.01923201    0.00000000   1       4  H
   -1.49611119   -4.60498625    0.00000000   2       5  N
   -0.94359104   -2.88324896   -0.00000001   3       6  Si
   -1.28979062   -0.69238461    0.00000001   3       7  Si
   -2.82615281    0.16010171    0.00000000   2       8  N
   -2.87378880    1.21247611    0.00000000   1       9  H
   -3.75689478   -0.33235037    0.00000000   1      10  H
    0.09887265    0.34342352    0.00000000   2      11  N
    1.75278495   -0.11954481   -0.00000001   3      12  Si
    2.70878199    1.03908226    0.00000000   1      13  H
    2.52893094   -1.67740697    0.00000000   2      14  N
    1.27270225   -2.87710973    0.00000000   3      15  Si

outcell: Unit cell vectors (Ang):
       14.782281    0.000000    0.000000
        0.000000   16.643462    0.000000
        0.000000    0.000000    9.854683

outcell: Cell vector modules (Ang)   :   14.782281   16.643462    9.854683
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :   2424.5312
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=155 nrows=48 sparsity=.3022 nnzs=7261, refcount: 8>
  <dData1D:(new from dSpData1D) n=7261, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=155 nrows=48 sparsity=.3022 nnzs=7261, refcount: 9>
  <dData2D:DM n=7261 m=8, refcount: 1>
refcount: 1>
     spin moment: S , {S} =   30.00000   0.00000   0.00000  30.00000
